cns/service/main.go

```diff
@@ -795,6 +801,35 @@ func main() {
 		}
 	}
 
+	type CNSService struct {
+		pb.UnimplementedCNSServiceServer
+		Logger *zap.Logger
+	}
```

Comment:
It's better if we don't define any stucts in the `main` function scope. Can this be moved to the `grpc` package?

Reply:
Yes, makes sense not sure why I missed that I have added that struct to the grpc package and removed it from main